apps/web/pages/auth/forgot-password/[id].tsx

```diff
@@ -63,7 +63,7 @@ export default function Page({ resetPasswordRequest, csrfToken }: Props) {
             </h2>
           </div>
           <p>{t("password_has_been_reset_login")}</p>
-          <Link href="/auth/login">
+          <Link href="/auth/login" passHref>
```

Comment:
These buttons don't receive `href` AFAIK

Reply:
They do: https://github.com/calcom/cal.com/blob/main/packages/ui/Button.tsx#L41

Reply:
You're referring to `Button` my comment refers to `button`s

Reply:
Gotcha, that's fixed